Tests/docker_native_image_config.json

```diff
@@ -29,7 +29,7 @@
             "chromium",
             "tesseract"
          ],
-         "docker_ref": "demisto/py3-native:8.1.0.40949"
+         "docker_ref": "demisto/py3-native:8.1.0.45852"
```

Comment:
this will be released under 8.1 also? We dont want to use a maintenance image here?

Reply:
why do you mean by "also" - this image replaced completely the old image in 8.1

Reply:
theres no rollover?

Reply:
as i said one this image is build, it completely replaces the old one.